ui/src/app/applications/components/application-details/application-details.tsx

```diff
@@ -866,10 +883,18 @@ export class ApplicationDetails extends React.Component<RouteComponentProps<{app
                                                 )}
                                         </SlidingPanel>
                                         <SlidingPanel
-                                            isShown={this.selectedExtension !== '' && activeExtension != null && activeExtension.flyout != null}
+                                            isShown={this.selectedExtension !== '' && activeStatusExt != null && activeStatusExt.flyout != null}
+                                            onClose={() => this.setExtensionPanelVisible('')}>
+                                            {this.selectedExtension !== '' && activeStatusExt && activeStatusExt.flyout && (
```

Comment:
```suggestion
                                            {this.selectedExtension !== '' && activeStatusExt?.flyout && (
```

same as `activeTopBarActionMenuExt` below, for consistency